versioned/persist/dynamodb/src/test/java/org/projectnessie/versioned/persist/dynamodb/ITDynamoDatabaseAdapter.java

```diff
@@ -16,7 +16,12 @@
 package org.projectnessie.versioned.persist.dynamodb;
 
 import org.projectnessie.versioned.persist.tests.AbstractDatabaseAdapterTest;
+import org.projectnessie.versioned.persist.tests.extension.NessieDbAdapterConfigItem;
 import org.projectnessie.versioned.persist.tests.extension.NessieExternalDatabase;
 
 @NessieExternalDatabase(LocalDynamoTestConnectionProviderSource.class)
+@NessieDbAdapterConfigItem(name = "retry.initial.sleep.millis.lower", value = "25")
```

Comment:
is the name intentionally different from the `retry.initial-sleep.millis-lower` in `QuarkusVersionStoreAdvancedConfig.java` ?

if yes what are the mapping rules?
if no, how do we prevent such errors from the future i.e. more type safety maybe?

Reply:
See DatabaseAdapterExtension